src/components/SpaceProposalBoost.vue

```diff
@@ -0,0 +1,386 @@
+        <div v-if="boosts.length">
+          <div class="md:flex md:justify-between">
+            <div>
+              <i-s-boost-logo class="text-xs text-skin-link" />
+              <p v-if="isActive" class="text-md leading-5 mt-2">
+                Get rewards by voting on this proposal
+              </p>
+            </div>
+            <TuneButton
+              v-if="isActive"
+              type="button"
+              class="flex items-center pl-[18px] pr-[22px] w-full md:w-auto justify-center mt-[12px] md:mt-0"
+              @click="handleBoost"
+            >
+              <i-ho-plus class="mr-2 text-xs" />
+              <span> New boost </span>
+            </TuneButton>
+          </div>
+          <LoadingList v-if="loading" class="mt-3" />
+          <div v-else>
+            <div class="mt-3 space-y-2">
+              <div
+                v-for="boost in showAllBoosts
+                  ? boostsSorted
+                  : boostsSorted.slice(0, INITIAL_VISIBLE_BOOSTS)"
+                :key="boost.id"
+              >
+                <SpaceProposalBoostItem
+                  :boost="boost"
+                  :claims="boostClaims"
+                  :proposal="proposal"
+                  :reward="
+                    boostRewards.find(
+                      reward =>
+                        reward.boost_id === boost.id &&
+                        reward.chain_id === boost.chainId
+                    )
+                  "
+                  :is-eligible="isEligible(boost)"
+                  @reload="loadBoosts()"
+                />
+              </div>
+            </div>
+            <TuneButton
+              v-if="boostsSorted.length > INITIAL_VISIBLE_BOOSTS"
+              class="w-full mt-3 flex items-center justify-center"
+              @click="showAllBoosts = !showAllBoosts"
+            >
+              <span v-if="showAllBoosts"> View less </span>
+              <span v-else> View more </span>
+              <i-ho-arrow-sm-down
+                class="ml-2"
+                :class="{ 'rotate-180': showAllBoosts }"
+              />
+            </TuneButton>
+          </div>
+          <div
+            v-if="eligibleBoosts.length && isActive"
+            class="bg-[--border-color-faint] border-t border-[--border-color-soft] -mx-3 -mb-3 mt-3 p-3"
+          >
+            <div class="md:flex md:items-center gap-3">
+              <div class="flex gap-[12px] items-center w-full">
+                <div
+                  class="h-[44px] w-[44px] flex-shrink-0 rounded-2xl shadow-xl"
+                >
+                  <div class="flex items-center justify-center h-full">
+                    <i-ho-gift class="text-skin-heading text-sm" />
+                  </div>
+                </div>
+                <div class="w-full">
+                  <h5 class="leading-none">Rewards</h5>
+                  <p class="leading-none">
+                    You’re eligible to {{ eligibleBoosts.length }} boost<span
```

Comment:
```suggestion
                    You're eligible to {{ eligibleBoosts.length }} boost<span
```

Wrong ASCII character for single quote